modules/migration-state-migration-cli.adoc

```diff
@@ -18,3 +19,39 @@ You can perform a state migration between clusters or within the same cluster.
 ====
 State migration migrates only the components that constitute an application's state. If you want to migrate an entire namespace, use stage or cutover migration.
 ====
+
+.Prerequisites
+
+* The state of the application on the source cluster is persisted in `PersistentVolumes` provisioned through `PersistentVolumeClaims`.
+
+* The manifests of the application are available in a central repository that is accessible from both the source and the target clusters.
+
+
+.Procedure
+
+. Migrate persistent volume data from the source to the target cluster.
++
+You can perform this step as many times as needed. The source application continues running.
+
+. Quiesce the source application.
++
+You can do this by setting the replicas of workload resources to 0, either directly on the source cluster or by updating the manifests in GitHub and re-syncing the ArgoCD application.
+
+. Clone application manifests to the target cluster.
++
+You can use ArgoCD to clone the application manifests to the target cluster.
+
+. Migrate the remaining volume data from the source to the target cluster.
++
+Migrate any new data created by the application during steps 1 and 2 by performing a final data migration.
```

Comment:
Consider if it's necessary to mention the specific steps. There's not a way to automatically sync these references, so it's possible that they could become inaccurate after future updates.

```suggestion
Migrate any new data created by the application during the migration process by performing a final data migration.
```